packages/platform-sdk/src/coins/network.models.ts

```diff
@@ -172,6 +174,7 @@ export interface CoinNetwork {
 	};
 	// @TODO: we could replace this with kebabCase(Object.keys(FeatureFlags.Transaction))
 	transactionTypes: CoinTransactionTypes[];
+	expirationType: ExpirationType;
```

Comment:
I would move this under the `crypto` key since it is related to when the network decides to reject this transaction.